packages/v2/contracts/ANTv2.sol

```diff
@@ -93,14 +89,30 @@ contract ANTv2 is IERC20 {
     }
 
     function _transfer(address from, address to, uint256 value) private {
-        require(to != address(this), "ANTV2:RECEIVER_IS_TOKEN");
+        require(to != address(this) && to != address(0), "ANTV2:RECEIVER_IS_TOKEN_OR_ZERO");
 
         // Balance is implicitly checked with SafeMath's underflow protection
         balanceOf[from] = balanceOf[from].sub(value);
         balanceOf[to] = balanceOf[to].add(value);
         emit Transfer(from, to, value);
     }
 
+    function getChainId() public pure returns (uint256 chainId) {
+        assembly { chainId := chainid() }
+    }
+
+    function getDomainSeparator() public view returns (bytes32) {
+        return keccak256(
+            abi.encode(
+                EIP712DOMAIN_HASH,
+                NAME_HASH,
+                VERSION_HASH,
+                getChainId(),
```

Comment:
@0xGabi coming back to https://github.com/aragon/aragon-network-token/pull/21#discussion_r502704704, it's worth mentioning that while I think the functionality proposed (being able to test a signature on testnets) can be useful, it introduces a security risk with replay-ability (as @eduadiez gratefully pointed out!).